src/rad/handler.clj

```diff
@@ -138,11 +149,13 @@
       :description (:description page)
       :level level
     }
-  )
-)
+  ))
+
+(defn ms-from-epoch []
+  (.getTime (new java.util.Date)))
 
 (defroutes app-routes
-  (GET "/api" [] (json/write-str (select-page pages)))
+  (GET "/api" [] (json/write-str (select-page (ms-from-epoch) (pages))))
```

Comment:
Feels like another hack, why not have the JS send an increasing number per request and mod that with page count to get an index?